Source/WebGPU/WGSL/Types.cpp

```diff
@@ -229,12 +229,12 @@ ConversionRank conversionRank(const Type* from, const Type* to)
     }
 
     if (auto* fromVector = std::get_if<Vector>(from)) {
-        auto* toVector = std::get_if<Vector>(to);
-        if (!toVector)
+        auto* vector = std::get_if<Vector>(to);
```

Comment:
Not sure this one should have been renamed. I suggest tossing the changes to this file.

Reply:
That was a auto replace mistake, will revert changes to this file